File: C/5.cpp
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>

int main(void){
	setlocale(LC_ALL, "Portuguese");
	float a,b,c;
	
	printf("Esse programa classifica se os valores formam um triangulo e determina qual o tipo!\n\n");
	
	printf("\n\nValor de X: ");
	scanf("%f",&a);
	
	printf("\n\nValor de Y: ");
	scanf("%f",&b);
	
	printf("\n\nValor de Z: ");
	scanf("%f",&c);
	
	if (a<=0 || b<=0 || c<=0){
	      printf("Lados nulos ou negativos nao sao aceitos."); 
	   }
	
	   if (a>=b+c || b>=c+a || c>=a+b) {
	      printf("Triangulo inexistente.");
	      
	   }
	
	   if (a==b && b==c)
	      printf("Triangulo equilatero.");
	
	   else if (a==b || b==c || c==a)
	      printf("Triangulo isosceles.");
	
	   else
	      printf("Triangulo escaleno.");
	
	
	printf("\n\nObrigado por usar nosso sistema!\n\n");
	system("pause");
	return 0;
}

File: Python/5.py
print("Esse programa classifica se os valores formam um triangulo e determina qual o tipo!\n\n");

a = float(input("\n\nValor de X: "))

b = float(input("\n\nValor de Y: "))

c = float(input("\n\nValor de Z: "))

if (a<=0 or b<=0 or c<=0):
	print("Lados nulos ou negativos nao sao aceitos.")

elif (a>=b+c or b>=c+a or c>=a+b):
	print("Triangulo inexistente.")

elif (a==b and b==c):
	print("Triangulo equilatero.")

elif (a==b or b==c or c==a):
	print("Triangulo isosceles.")

else:
	print("Triangulo escaleno.")


print("\n\nObrigado por usar nosso sistema!\n\n")
